packages/core/addon/helpers/smart-format-number.js

```diff
@@ -30,7 +31,7 @@ export function smartFormatNumber([value]) {
   let absValue = Math.abs(value);
 
   if (absValue === 0 || (absValue >= 1 && absValue < 100)) {
-    return numeral(value).format('0,0.00');
+    return numeral(value).format(hasPoint ? '0,0.00' : '0,0[.]00');
```

Comment:
Does fili always return either an integer or a float? if it sends a mixture of integers or floats, how would this work?

Reply:
So each cell will make it's own determination of the format.  So you can see columns formatted differently for each row if the webservice returns a mix of both.  But this puts some responsibility on the web service to return floats if the type is floaty consistently.